Services/CO.CDP.OrganisationInformation/BuyerInformation.cs

```diff
@@ -0,0 +1,10 @@
+namespace CO.CDP.OrganisationInformation;
+
+public record BuyerInformation
+{
+
+    public string? BuyerType { get; set; }
+
+    public List<DevolvedRegulation> DevolvedRegulations { get; set; } = [];
+    
+}
```

Comment:
This type isn't meant to be shared between APIs, and should go to the Organisation Web API. 

To avoid polluting this common namespace, it's best to start with types kept in the Web API they originated and only move to CO.CDP.OrganisationInformation through refactoring when they're needed to be shared.